db/migrate/20230124180818_create_comments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.references :author, null: false, foreign_key: true
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] Please make sure that the comment has a foreign key reference to the posts table, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) ERD Diagram.